instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/jvm/JvmMetricsCollector.scala

```diff
@@ -259,4 +268,27 @@ object JvmMetricsCollector {
     def sanitize(name: String): String =
       _invalidChars.replaceAllIn(name.toLowerCase, "-")
   }
+  
+  sealed trait ThreadState
+  object ThreadState {
+    case object New extends ThreadState { override def toString: String = "new"}
+    case object Runnable extends ThreadState { override def toString: String = "runnable" }
+    case object Blocked extends ThreadState { override def toString: String = "blocked" }
+    case object Waiting extends ThreadState { override def toString: String = "waiting" }
+    case object TimedWaiting extends ThreadState { override def toString: String = "timed_waiting" }
```

Comment:
Can you please change `timed_waiting` with `timed-waiting`. This way the metric would match exactly the same output as the corresponding metric in Micrometer :smile:

Reply:
Thanks for review. I've changed it.